models/lido/liquidity/polygon/lido_liquidity_polygon_balancer_pools.sql

```diff
@@ -0,0 +1,551 @@
+{{ config(
+    schema='lido_liquidity_polygon', 
```

Comment:
does this new model have the same issues as the other model we're tracking/fixing?

Reply:
yep, it has the same problem, let me change to table